packages/ckeditor5-build-classic/src/ckeditor.js

```diff
@@ -101,7 +115,8 @@ ClassicEditor.defaultConfig = {
 			'htmlEmbed',
 			'mediaEmbed',
 			'|',
-			'uploadImage'
+			'uploadImage',
+			'emoji'
```

Comment:
Is it so hard to confirm once with me with the screenshot the placement of the button on Slack? 

Lets put blockQuote after link. And put emoji after indent so `outdent indent | emoji | htmlEmbed mediaEmbed...`